src/ServicePulse.Host/app/js/services/services.service-control.js

```diff
@@ -221,6 +221,13 @@
                 });
         }
 
+        function acknowledgeArchiveGroup(groupId, successText, failureText) {
```

Comment:
from what I see successText and failureText is not being used in this code. Please remove those parameters